docs/UserGuide.md

```diff
@@ -303,7 +303,23 @@ bye
 
 ### 5.3. Students
 
-[short overview]
+{ start of `Features#Students` written by: Zhang Sheng Yang }
```

Comment:
Perhaps `Features#Students` could be renamed to `features#students`? Same comment for line 322.